python/core/auto_generated/symbology/qgsstyle.sip.in

```diff
@@ -466,12 +467,11 @@ Is emitted every time a tag or smartgroup has been added, removed, or renamed
 Convenience function to open the DB and return a sqlite3 object
 %End
 
-    bool runEmptyQuery( char *query, bool freeQuery = true );
+    bool runEmptyQuery( const QString &query );
```

Comment:
I don't think this should be exposed to SIP. In fact, all these members should be private and not protected (This class shouldn't be subclassed). (I suspect this dates from some time in ancient QGIS history where the default behaviour was to use protected for everything instead of private).